contracts/market/src/state/spot_price.rs

```diff
@@ -653,12 +656,19 @@ impl State<'_> {
                                     },
                                 );
                             } else if let Entry::Vacant(entry) = rujira.entry(asset.clone()) {
-                                let pool = rujira_rs::query::Pool::load(
```

Comment:
We should be using the enshrined oracle for _all_ price feeds, including RUNE.

Can we ensure that we maintain support for the current price feed identifiers? In other words, I'd like to be able to migrate to the new contract, keep the spot price config of e.g. `BTC.BTC`, and have everything continue working.